.github/workflows/ci-zod-openapi.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         with:
           node-version: 20.x
       - run: yarn workspaces focus hono-middleware @hono/zod-openapi
-      - run: yarn workspace @hono/zod-openapi build
+      - run: yarn workspaces foreach --topological --recursive --from @hono/zod-openapi run build
```

Comment:
Recursively builds the dependencies of the current workspace

This is specifically for `@hono/zod-openapi` as it has a dependency on `@hono/zod-validator` which needs building first